csharp/src/SeedLang/Visualization/IVM.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace SeedLang.Visualization {
+  // The interface of SeedLang VM. The methods of it can only be called during visualization
+  // notification.
+  public interface IVM {
+    // Stops execution.
+    void Stop();
```

Comment:
1. The actual meaning of this method is `Pause`, right? `Pause` is a better name than `Stop` here.

2. We also need a real `Stop` or `Quit` interface. This can be provided either in the IVM interface, or in the Engine class.

Reply:
Changed to `Pause` and added a real `Stop` interface.